src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -360,22 +457,109 @@ ApiGlobe.prototype.createSceneGlobe = function createSceneGlobe(coordCarto, view
+    this.sceneLoadedDeferred = defer();
+    this.addEventListener('globe-loaded', () => {
+        this.sceneLoadedDeferred.resolve();
+        this.sceneLoadedDeferred = defer();
+    });
 
-    this.scene.add(map);
+    // init globe layer with default parameter
+    const wgs84TileLayer = {
+        protocol: 'tile',
+        id: globeLayerId,
+        update: (context, layer, node) => updateTreeLayer(processTiledGeometryNode, context, layer, node),
+        nodeType: TileMesh,
+        maxLevel: 16,
+        ellipsoid,
+        initRootNodes: initTiledGeometryLayer,
+        schemeTile: globeSchemeTileWMTS(globeSchemeTile1),
+        cullingTest: globeCulling,
+        preUpdate: (context, layer) => preGlobeUpdate(context, layer),
+        initNewNode,
+        mustSubdivide: globeSubdivisionControl,
+        builder: new BuilderEllipsoidTile(ellipsoid, new Projection()),
+        // TODO: add a layer to be able to use Camera.layers feature
+    };
+
+    const atmosphere = new Atmosphere(ellipsoid);
+    atmosphere.add(new Clouds());
+    this.scene.gfxEngine.scene3D.add(atmosphere);
+
+    const debugIdUpdate = function debugIdUpdate(context, layer, node) {
+        var n = node.children.filter(n => n.layer == debugLayer.id);
+        if (node.material.visible) {
+            if (n.length == 0) {
+                n = new OBBHelper(node.OBB(), `id:${node.id}`);
+                n.layer = debugLayer.id;
+                node.add(n);
+                n.update(node.OBB());
+            } else {
+                n = n[0];
+            }
+            n.setMaterialVisibility(true);
+        }
+        if (n.length > 0) {
+            n[0].setMaterialVisibility(false);
+        }
+    };
+
+    const debugLayer = {
+        protocol: 'dummy',
+        id: 'tile_ids',
+        update: debugIdUpdate,
+        graftedOnId: globeLayerId,
+    };
+
+    // uncomment next line to display boundingbox helpers drawn
+    // this.addGeometryLayer(debugLayer);
+
+    //
+    // Create Control
+    //
+    this.scene.controls = new GlobeControls(this.scene.camera.camera3D, this.scene.gfxEngine.renderer.domElement, this.scene.gfxEngine);
+    this.scene.controls.rotateSpeed = 0.25;
+    this.scene.controls.zoomSpeed = 2.0;
+    this.scene.controls.minDistance = 30;
+    this.scene.controls.maxDistance = ellipsoid.size.x * 8.0;
+
+    this.scene.controls.addEventListener('change', this.scene.gfxEngine.update);
+
+    return wgs84TileLayer;
+};
 
-    // Register all providers
-    var wmtsProvider = new WMTS_Provider({
-        support: map.gLDebug,
-    });
 
-    this.scene.scheduler.addProtocolProvider('wmts', wmtsProvider);
-    this.scene.scheduler.addProtocolProvider('wmtsc', wmtsProvider);
-    this.scene.scheduler.addProtocolProvider('tile', new TileProvider(ellipsoid));
-    this.scene.scheduler.addProtocolProvider('wms', new WMS_Provider({ support: map.gLDebug }));
+ApiGlobe.prototype.createScene = function createScene(viewerDiv, bbox) {
```

Comment:
The shared code between `createScene*` methods should probably go in Scene's constructor or a `init` method.
And `createScene` should be removed from ApiGlobe because it's not globe-specific.